UDF/README.UDF

```diff
@@ -0,0 +1,8 @@
+README for Algorithms UDF Directory
+
+Some algorithms require the use of User Defined Functions :
+  KMeans - rand_int()
+
+Please place the ExprFunctions.hpp file into this directory on your TigerGraph installation system :
+
+~/tigergraph/app/3.1.0/dev/gdk/gsql/src/QueryUdf/
```

Comment:
Might be better to say
~/tigergraph/app/<version>/dev/gdk/gsql/src/QueryUdf/
so that this README file doesn't automatically became outdated when 3.2.0.